packages/filter/src/features/search-filter/section-factories/boolean-record/BooleanRecordChips.ts

```diff
@@ -46,6 +46,7 @@ export const createOnClickRemoveOnChipForBooleanRecord = <
 ): SearchFilterSectionOnClickRemoveOnChip<TFormModel> => {
   return ({ setFormModelFields, value }) => {
     const newRecord = { ...formModel[fieldName] };
+    // @ts-ignore
```

Comment:
Can we avoid using `@ts-ignore`?

Reply:
https://www.typescriptlang.org/tsconfig/#suppressImplicitAnyIndexErrors

![image](https://github.com/StenaIT/stenajs-webui/assets/1266041/6ccb4dae-562a-4947-8454-18a212573cd2)

:P

Reply:
I fixed it!